spec/unit/mutant/reporter/null_spec.rb

```diff
@@ -2,21 +2,11 @@
   let(:object) { described_class.new     }
   let(:value)  { instance_double(Object) }
 
-  describe '#report' do
-    subject { object.report(value) }
+  %i[progress report start warn].each do |name|
+    describe "##{name}" do
+      subject { object.public_send(name, value) }
 
-    it_should_behave_like 'a command method'
-  end
-
-  describe '#warn' do
-    subject { object.warn(value) }
-
-    it_should_behave_like 'a command method'
-  end
-
-  describe '#progress' do
-    subject { object.progress(value) }
-
-    it_should_behave_like 'a command method'
+      it_should_behave_like 'a command method'
+    end
```

Comment:
There is no to DRY, my code is ideally reduced to desert. RSpec triggers too redundant syntax anyway. The shorter I can get it the easier to maintain.

This was triggered by the fact I had to copy and paste a block to cover a missed method, while just changeing the message being send to the object. Copy and pasting something that is by 95% equivalent to 3 other instances is not a development / maintenance strategy I like.

The DRY-er the better.